src/routes/login.ts

```diff
@@ -4,7 +4,9 @@ import { createUser, fetchUser } from '../lib/database/users'
 import { createSession } from '../lib/database/sessions'
 import type { RequestHandler } from '@sveltejs/kit'
 import { createUuid } from '../lib/database/index'
-import config from '../lib/config'
+
+const clientID = process.env['DISCORD_CLIENT_ID']
```

Comment:
This variable should be called `clientId` instead of `clientID`. Also this should use dot notation instead of bracket notation.

Reply:
This issue has been fixed